### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by theVivek-kumar**
wish list done please review

### CODE REVIEW

File: my-cart/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+body {
+    background-color: #ebebeb;
+}
+
+:root {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    --font-color: #282a36;
+    --BACKGROUND-color: #f39494;
+    --PRIMERY-color: #312546;
```
Comment (Ananyamadhu08) [inline my-cart/wishlist/wishlist.css]: there is a typo here, primary.

File: my-cart/wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+.navbar-icons {
+    display: flex;
+    justify-content: space-around;
+
+
+}
+
+.fa-search {
+    padding: 4px;
```
Comment (Ananyamadhu08) [inline my-cart/wishlist/wishlist.css]: use rem instead of px, use consistent units.

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go, can go ahead and merge 👍

File: my-cart/mycart.css [hunk 3]
```diff
@@ -23,6 +23,10 @@ body {
     height: auto;
 
 }
+.navbar-head-logo a{
+    text-decoration: none;
+    color: #ece9f2;
```
Comment (deekshithmd) [inline my-cart/mycart.css:28]: Better to use color variable

File: my-cart/wishlist/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+.fa-heart {
+    margin: 1rem;
+    font-size: 1.3rem;
+    color: red;
+}
+
+.search-button {
+    border: none;
+    border-radius: 5px 5px 5px 5px;
```
Comment (deekshithmd) [inline my-cart/wishlist/wishlist.css:94]: if you need same border radius in all 4 corner, then you can give one value

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**